weather-app/src/tests/components/ForecastSummaries.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummaries from "../../components/ForecastSummaries";
+
+describe("ForecastSummaries", () => {
+  const validProps = [
+    {
+      date: 11100011,
+      description: "Stub description 1",
+      icon: 500,
+      temperature: {
+        min: 12,
+        max: 22,
+      },
+    },
+    {
+      date: 22200011,
+      description: "Stub description 2",
+      icon: 600,
+      temperature: {
+        min: 15,
+        max: 27,
+      },
+    },
+  ];
+
+  it("renders correctly", () => {
+    const { asFragment } = render(<ForecastSummaries forecasts={validProps} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
So theres a few things that need to be fixed in this test. First off, you are missing a required prop for the component, onForecastSelect.

This should be added to the validProps object and referenced in the component instantiations to get rid of the console errors

Also, the structure of the valid props will need updating so it is an object with the properties of `forecasts` and `onForecastSelect`

Once this has been done, you will need to update the component instantiation in the it blocks like so:
```
  const validProps = {
    forecasts: [
      {
        date: 11100011,
        description: "Stub description 1",
        icon: 500,
        temperature: {
          min: 12,
          max: 22,
        },
      },
      {
        date: 22200011,
        description: "Stub description 2",
        icon: 600,
        temperature: {
          min: 15,
          max: 27,
        },
      },
    ],
    onForecastSelect: jest.fn(),
  };

  it("renders correctly", () => {
    const { asFragment } = render(
      <ForecastSummaries
        forecasts={validProps.forecasts}
        onForecastSelect={validProps.onForecastSelect}
      />,
    );
```